ResidentInformationApi/V1/Factories/ResponseFactory.cs

```diff
@@ -1,55 +1,197 @@
+using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
 using ResidentInformationApi.V1.Boundary.Responses;
 using ResidentInformationApi.V1.Domain;
+using AcademyClaimantInformation = ResidentInformationApi.V1.Domain.AcademyClaimantInformation;
+using AcademyClaimantInformationResponse = ResidentInformationApi.V1.Boundary.Responses.AcademyClaimantInformation;
+using AcademyClaimantInformationResult = ResidentInformationApi.V1.Domain.AcademyClaimantInformationResult;
+using AcademyClaimantInformationResultResponse = ResidentInformationApi.V1.Boundary.Responses.AcademyClaimantInformationResult;
 using Address = ResidentInformationApi.V1.Domain.Address;
 using AddressResponse = ResidentInformationApi.V1.Boundary.Responses.Address;
+using Email = ResidentInformationApi.V1.Domain.Email;
+using EmailResponse = ResidentInformationApi.V1.Boundary.Responses.Email;
 using HousingResidentInformation = ResidentInformationApi.V1.Domain.HousingResidentInformation;
 using HousingResidentInformationResponse = ResidentInformationApi.V1.Boundary.Responses.HousingResidentInformation;
+using HousingResidentInformationResult = ResidentInformationApi.V1.Domain.HousingResidentInformationResult;
+using HousingResidentInformationResultResponse = ResidentInformationApi.V1.Boundary.Responses.HousingResidentInformationResult;
+using MosaicResidentInformation = ResidentInformationApi.V1.Domain.MosaicResidentInformation;
+using MosaicResidentInformationResponse = ResidentInformationApi.V1.Boundary.Responses.MosaicResidentInformation;
+using MosaicResidentInformationResult = ResidentInformationApi.V1.Domain.MosaicResidentInformationResult;
+using MosaicResidentInformationResultResponse = ResidentInformationApi.V1.Boundary.Responses.MosaicResidentInformationResult;
+using PhoneType = ResidentInformationApi.V1.Domain.PhoneType;
+using PhoneTypeResponse = ResidentInformationApi.V1.Boundary.Responses.PhoneType;
 
 namespace ResidentInformationApi.V1.Factories
 {
     public static class ResponseFactory
     {
+        public static ContactInformationResponse ToResponse(this ContactInformation domain)
+        {
+            return new ContactInformationResponse
+            {
+                AcademyResult = domain.AcademyResult.ToResponse(),
```

Comment:
Is `domain.AcademyResult` not also nullable?

Reply:
Yes, it is, I'll update that